src/main/java/subsystems/Leds/Leds.java

```diff
@@ -0,0 +1,36 @@
+package subsystems.Leds;
+
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.AddressableLED;
+
+
+
+public class Leds extends SubsystemBase {
+   public static Leds INSTANCE;
+
+    public static Leds getINSTANCE() {
+        if(INSTANCE == null){
+            return INSTANCE;
+        }
+        return null;
+    }
+
+    public AddressableLED leds = new AddressableLED(0);
+   public AddressableLEDBuffer ledBuffer = new AddressableLEDBuffer(LedConstants.LED_LENGTH);
+
+   public Leds(){
+       leds.setLength(ledBuffer.getLength());
+   }
+   public void setYellow(){
+        for(int i = 0; i<ledBuffer.getLength(); i++ ){
+            ledBuffer.setRGB(i,254, 202, 82);
+        }
+    }
+    public void setPurple(){
+        for(int i = 0; i<ledBuffer.getLength(); i++ ){
+            ledBuffer.setRGB(i,51, 1, 176);
+        }
+    }
```

Comment:
1. There is some repetition between these two methods. What can you do to reduce repetition?
2. [This](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/AddressableLEDBuffer.html#setLED(int,edu.wpi.first.wpilibj.util.Color)) specific method might come handy 🙂.

Reply:
Again, why did you resolve the conversation? You didn't address the first note and half-addressed the second one.